### DISCUSSION TITLE
Treat single-value enums as constants

### DISCUSSION THREAD

**Discussion by federicobond** [scalar/scalar#1837] [Ideas]
Since OpenAPI 3.0 does not have a way of defining constant property values, it's common practice to use single-value enums to represent such values.

The Scalar UI currently treats these the same as other enums but it would be better for readers if it was rendered in a way that made that distinction clear.

Check out this discussion for reference: https://github.com/OAI/OpenAPI-Specification/issues/1313

OpenAPI 3.1 appears to add support for `const` directly, which would also be a nice addition to Scalar too.

**Comment by marclave** [comment]
ah this is a great idea! let's create an issue :)
